### PR TITLE
Some improvements to mathematical style.

### PR DESCRIPTION
**Pull request by SouthEndMusic**
Some changes to the mathematical notation to adhere to conventions, e.g. only variables in italics and descriptive subscripts with normal text.

### CODE REVIEW

File: docs/core/equations.qmd [hunk 1]
```diff
@@ -28,35 +28,35 @@ relationships.
...
-Q_E = E_{pot} \cdot A(S) \cdot (0.5 \cdot \tanh(S - 50.0) / 10.0) + 0.5
+Q_E = E_\text{pot} \cdot A(S) \cdot \frac{1}{2}\left( \tanh\left(\frac{S-50}{10}\right) +1\right)
 $$
 
-Where $E_{pot}$ is the potential evaporation rate and $A$ is the wetted area.
-The $(0.5 \cdot \tanh(S - 50.0) / 10.0) + 0.5$ term ensures no evaporation occurs
-when $S <= 0$ and is a sigmoid function which provides a smooth gradient as $S
+Here $E_\text{pot}$ is the potential evaporation rate and $A$ is the wetted area.
+The $\frac{1}{2}\left( \tanh\left(\frac{S-50}{10}\right) +1\right)$ term ensures almost no evaporation occurs
+when $S \le 0$ and is a sigmoid function which provides a smooth gradient as $S
```
Comment (Hofer-Julian) [inline docs/core/equations.qmd:48]: What does `\le` change?

File: docs/core/equations.qmd [hunk 2]
```diff
@@ -28,35 +28,35 @@ relationships.
...
+function of the storage $S$: as the volume of water changes, the area of the free water
 surface may change as well, depending on the slopes of the surface waters. 
 
 ## Evaporation
 
 The evaporation term is given by:
 
 $$
-Q_E = E_{pot} \cdot A(S) \cdot (0.5 \cdot \tanh(S - 50.0) / 10.0) + 0.5
+Q_E = E_\text{pot} \cdot A(S) \cdot \frac{1}{2}\left( \tanh\left(\frac{S-50}{10}\right) +1\right)
```
[the remark this answers is not in the document]
Comment (visr) [reply docs/core/equations.qmd:43]: The tanh is currently not used, this equation is outdated. We currently use a reduction factor as follows:
```julia
reduction_factor = min(depth, 0.1) / 0.1
evaporation = area * reduction_factor * basin.potential_evaporation[i](t)
```
Would be good to directly update the docs to that.

File: docs/core/equations.qmd (hunk 1 above)
Comment (SouthEndMusic) [reply docs/core/equations.qmd:48]: `\le` contracts `<=`  into a single symbol, it stands for lesser than or equal to (it is equivalent to `\leq`).

**Comment by SouthEndMusic** [comment]
> Looks good to me. Did you render the docs locally to check if your changes are fine?

Yes, I use the Quarto extension for VScode.

**Pull request merged**